doc/05-Howto-Jetstream.md

```diff
@@ -45,12 +45,12 @@ For now, click on __Launch New Instance__ and type __bgRAMOSE__ into the search
 Click on the top icon that comes up, corresponding to the latest version, read the description, and click __Launch__.
 Edit the __Instance Name__ to your liking, select _m1.medium_ under __Instance Size__, and launch the instance.
 Your screen will show the progress of the VM deployment.
-You are ready for the next step once the screen indicates status __Active__ and provides an IP number.
+You are ready for the next step once the screen indicates status __Active__ and provides an IP address.
```

Comment:
IP address is more common. IP number isn't necessarily wrong, but might confuse a newcomer.